### PR TITLE
Add support for displaying extra help

### PR DESCRIPTION
**Pull request by dlaliberte**
This PR uses sl-details to display extra help for form fields, if extra_help field is specified.  The style looks the same with and without extra help, with the addtion of a ">" icon to show the extra help.

### CODE REVIEW

**Comment by dlaliberte** [comment]
I converted part of the help text for "Requires Embedder Support" to extra_help, just as an example. 

image

image

File: static/elements/chromedash-form-field.js [hunk 1]
```diff
@@ -70,7 +99,16 @@ export class ChromedashFormField extends LitElement {
         </td>
         <td>
           <span class="helptext">
-            ${helpText}
+            ${!extraHelpText ? helpText :
+              html`
+              <sl-details>
+                <span slot="summary">
+                  ${helpText}
```
[the remark this answers is not in the document]
Comment (dlaliberte) [reply static/elements/chromedash-form-field.js:106]: Yes, it works the same as any block element.

**Comment by dlaliberte** [comment]
Here is what it looks like with the extra help moved to a new row:

**Comment by dlaliberte** [comment]
Here is what the current PR looks like, with the + and - icons.

image

**Review by jrobbins — APPROVED** [review]

**Pull request merged**